master/buildbot/test/unit/test_stats_service.py

```diff
@@ -313,7 +313,7 @@ def test_build_start_time_capturing(self):
         yield self.end_build_call_consumers()
 
         self.assertEqual(
-            'start-time', self.fake_storage_service.stored_data[0][0].keys()[0])
```

Comment:
I think, for first item of the key generator, its best to just use

    future.util.iterkeys(self.fake_storage_service.stored_data[0][0]).next()

instead of transforming it to a list